packages/@expo/cli/src/utils/errors.ts

```diff
@@ -60,7 +60,10 @@ export function logCmdError(error: Error): never {
     exit(error);
   }
 
-  exit(chalk.red(error.toString()) + '\n' + chalk.gray(error.stack));
+  // @ts-ignore note(simek): Ignore unknown `isConfigError` field to avoid importing `ConfigError` class (CLI perf).
+  const errorDetails = error.stack && !error?.isConfigError ? '\n' + chalk.gray(error.stack) : '';
+
+  exit(chalk.red(error.toString()) + errorDetails);
```

Comment:
That's good point, thank you! I think that we can also relay on `error.name === 'ConfigError'` instead of `isConfigError` field and suppressing TS warning. We set this name for all config errors, wdyt?

https://github.com/expo/expo/blob/082815dcf3e2de29c76f7628844ae50a987263a6/packages/%40expo/config/build/Errors.d.ts#L8